src/app/tests/suites/certification/Test_TC_MF_1_10.yaml

```diff
@@ -18,63 +18,407 @@ name:
-
-          ./chip-tool administratorcommissioning  open-basic-commissioning-window 500 1 0
-
-
-          [1635871562.958905][4336:4341] CHIP:DMG: InvokeCommand =
-          [1635871562.958976][4336:4341] CHIP:DMG: {
-          [1635871562.959031][4336:4341] CHIP:DMG:         CommandList =
-          [1635871562.959095][4336:4341] CHIP:DMG:         [
-          [1635871562.959161][4336:4341] CHIP:DMG:                 CommandDataIB =
-          [1635871562.959225][4336:4341] CHIP:DMG:                 {
-          [1635871562.959290][4336:4341] CHIP:DMG:                         CommandPathIB =
-          [1635871562.959368][4336:4341] CHIP:DMG:                         {
-          [1635871562.959434][4336:4341] CHIP:DMG:                                 EndpointId = 0x0,
-          [1635871562.959530][4336:4341] CHIP:DMG:                                 ClusterId = 0x3c,
-          [1635871562.959625][4336:4341] CHIP:DMG:                                 CommandId = 0x1,
-          [1635871562.959698][4336:4341] CHIP:DMG:                         },
-          [1635871562.959776][4336:4341] CHIP:DMG:
-          [1635871562.959841][4336:4341] CHIP:DMG:                         StatusIB =
-          [1635871562.959911][4336:4341] CHIP:DMG:                         {
-          [1635871562.959980][4336:4341] CHIP:DMG:                                 status = 0x0,
-          [1635871562.960064][4336:4341] CHIP:DMG:                         },
-          [1635871562.960313][4336:4341] CHIP:DMG:
-          [1635871562.960459][4336:4341] CHIP:DMG:                 },
-          [1635871562.960544][4336:4341] CHIP:DMG:
-          [1635871562.960604][4336:4341] CHIP:DMG:         ],
-          [1635871562.960668][4336:4341] CHIP:DMG:
-          [1635871562.960723][4336:4341] CHIP:DMG: }
-          [1635871562.960824][4336:4341] CHIP:DMG: Received Command Response Status for Endpoint=0 Cluster=0x0000_003C Command=0x0000_0001 Status=0x0
-          [1635871562.960894][4336:4341] CHIP:TOO: Default Success Response
-      disabled: true
-
-    - label:
-          "Set up a TH_CR2 to start attempting to do PASE to DUT_CE and failing
-          20 times. This can be done using an invalid passcode"
-      verification: |
-          On 2nd controller, using chip-tool connect using wrong passcode. Repeat the step for 20 times.
-
-          ./chip-tool pairing onnetwork 1 20202019  (With wrong passcode)
-      disabled: true
-
-    - label: "TH_CR2 starts a 21st commissioning process"
-      verification: |
-          On 2nd controller, using chip tool connect using correct passcode.
-
-          ./chip-tool pairing onnetwork 1 20202021 (correct passcode)
-      disabled: true
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "TH_CR1 opens a commissioning window on DUT_CE"
```

Comment:
This test doesn't appear to be doing the correct steps in [MF_1_10](https://github.com/CHIP-Specifications/chip-test-plans/blob/master/src/multiplefabrics.adoc#3110-tc-mf-110-device-exit-commissioning-mode-after-20-failed-commission-attempts-bcm-dut---commissionee).

Reply:
Hi josh, i was reviving your comment on MF-1-10, was wondering if there was anything specific that you thought that did not match the test plan.

Reply:
I misread the YAML.